### PR TITLE
doc: releases: Add CVE-2019-9506 to release notes

### PR DESCRIPTION
**Pull request by d3zd3z**
Signed-off-by: David Brown <david.brown@linaro.org>

### CODE REVIEW

**Review by dbkinder — APPROVED** [review]
LGTM, thanks.

**Review by dleach02 — APPROVED** [review]

**Comment by ioannisg** [comment]
@d3zd3z this should not pass the check; commit message has no body :)

File: doc/releases/release-notes-2.0.rst [hunk 1]
```diff
@@ -23,6 +23,19 @@ Major enhancements with this release include:
 
 The following sections provide detailed lists of changes by component.
 
+Security Vulnerability Related
+******************************
+
+The following security vulnerability (CVE) was address in this
```
Comment (ioannisg) [inline doc/releases/release-notes-2.0.rst]: was addressed*

**Review by ioannisg — COMMENTED** [review]
Fix commit message and a minor grammar issue

**Comment by d3zd3z** [comment]
Any recommendations on what meaningful to put in the body of the commit? I'm normally for having a descriptive commit text, but the commit is just a small paragraph of prose, and it is kind of silly to be redundantly put that text into the commit text.

**Comment by ioannisg** [comment]
`Add section about security vulnerability issues in the 2.0.0 release notes.`

**Review by dleach02 — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**